.travis.yml

```diff
@@ -89,25 +89,24 @@ install:
 # i.e. There should be no DownloadWarning reports in the log.
   - python -c 'import cartopy; cartopy.io.shapereader.natural_earth()'
 
-# iris test data
+# download iris test data
   - >
     if [[ "$TEST_MINIMAL" != true ]]; then
       wget --quiet -O iris-test-data.zip https://github.com/SciTools/iris-test-data/archive/${IRIS_TEST_DATA_REF}.zip;
       unzip -q iris-test-data.zip;
       mv "iris-test-data-${IRIS_TEST_DATA_SUFFIX}" iris-test-data;
     fi
 
-# set config paths
-  - >
-    SITE_CFG=lib/iris/etc/site.cfg;
-    echo "[Resources]" > $SITE_CFG;
-    echo "test_data_dir = $(pwd)/iris-test-data/test_data" >> $SITE_CFG;
-    echo "doc_dir = $(pwd)/docs/iris" >> $SITE_CFG;
```

Comment:
The `doc_dir` is still referenced within `iris/tests/runner/_runner.py` and `iris/tests/test_coding_standards.py`